tvb_library/tvb/basic/neotraits/_declarative_base.py

```diff
@@ -31,10 +31,17 @@
 import abc
 import inspect
 import typing
+
 from tvb.basic.neotraits.ex import TraitTypeError, TraitAttributeError
-from tvb.basic.neotraits.info import auto_docstring
+from tvb.basic.neotraits.info import auto_docstring, prepare_html
 from tvb.basic.logger.builder import get_logger
 
+try:
+    from IPython.display import display_html
+except ImportError:
+    def display_html(doc, raw):
+        return doc
+
```

Comment:
This has a nice outcome, but I am not convinced the ipython dependency is healthy, being here at the very core.
![image](https://user-images.githubusercontent.com/674853/213639536-8185aaef-2af3-467c-8476-fc93fe461d37.png)
@maedoc @i-Zaak what do you think ?